docs/functions.rst

```diff
@@ -1572,6 +1572,74 @@ Example:
         args:
           name: "test-snapshot-content-content-dfc8fa67-8b11-4fdf-bf94-928589c2eed8"
 
+.. _restoredatausingkopiaserver:
+
+RestoreDataUsingKopiaServer
+---------------------------
+
+This function restores data backed up by the ``BackupDataUsingKopiaServer`` function.
+It creates a new Pod that mounts the PVCs referenced by the specified Pod
+and restores data to the specified path.
+
+.. note::
+   It is extremely important that, the PVCs are not be currently
+   in use by an active application container, as they are required
+   to be mounted to the new Pod (ensure by using
+   ``ScaleWorkload`` with replicas=0 first).
+   For advanced use cases, it is possible to have concurrent access but
+   the PV needs to have ``RWX`` mode enabled and the volume needs to use a
+   clustered file system that supports concurrent access.
+
+.. csv-table::
+   :header: "Argument", "Required", "Type", "Description"
+   :align: left
+   :widths: 5,5,5,15
+
+   `namespace`, Yes, `string`, namespace of the application that you want to restore the data in
+   `backupIdentifier`, Yes, `string`, unique snapshot id generated during backup
+   `restorePath`, Yes, `string`, path where data to be restored
+   `pod`, No, `string`, pod to which the volumes are attached
+   `volumes`, No, `map[string]string`, Mapping of `pvcName` to `mountPath` under which the volume will be available
+   `podOverride`, No, `map[string]interface{}`, specs to override default pod specs with
+
+.. note::
+   Between the ``pod`` and ``volumes`` arguments, exactly one argument
+   must be specified.
```

Comment:
```suggestion
   Either ``pod`` or the ``volumes`` arguments must be specified to this function based on the function that was used to backup the data. For `BackupData` we should specify `pod` and for `CopyVolumeData`, `Volumes` should be specified.
```

Feel free to change words/grammar according to you.

Reply:
LGTM